<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFilterConfig</class>
 <widget class="QFrame" name="frmFilterConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>AUX Channels</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>High Cutoff [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Low Cutoff [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Notchfilter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBoxEegNotch">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>on</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxEegLowpass"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxEegHighpass"/>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>EEG Channels</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>[Hz]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_Notch">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="maxVisibleItems">
        <number>2</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Notchfilter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
